src/module/Normalizer/src/Normalizer/Factory/NormalizeHelperFactory.php

```diff
@@ -38,7 +38,10 @@ public function createService(ServiceLocatorInterface $serviceLocator)
     {
         $normalize  = new Normalize();
         $normalizer = $serviceLocator->getServiceLocator()->get('Normalizer\Normalizer');
+        $instanceManager = $serviceLocator->getServiceLocator()->get('Instance\Manager\InstanceManager');
```

Comment:
added by @inyono: pass `InstanceManager` to `Normalizer` so we get a tenant-aware open search